gen_wiki.py

```diff
@@ -0,0 +1,616 @@
+            if exists(folder_name + folder):
+                if not isdir(folder_name + folder):
+                    if verbose:
+                        print("Does not exist. Making...")
+                    os.mkdir(folder_name + folder)
+                else:
+                    if verbose:
+                        print("Exists. Deleting and making new...")
+            else:
+                os.mkdir(folder_name + folder)
+                if verbose:
+                    print("Does not exist. Making...")
+
+        if quite or verbose:
+            print("==> Adding page start...")
+        if verbose:
+            print(" -> Adding index...")
+        index: str = Wiki.start()
+        if verbose:
+            print(" -> Adding overview...")
+        index += Wiki.overview(multi_page=True)
+        index += "\n---\n"
+        if verbose:
+            print(" -> Adding table of contents...")
+        index += Wiki.table_of_contents(multi_page=True)
+        if verbose:
+            print(f" -> Writing to \"{folder_name}/index.md\"...")
+        with open(f"{folder_name}/index.md", 'w') as file:
+            file.write(index)
+
+        if quite or verbose:
+            print("==> Adding poketes...")
+        if verbose:
+            print(" -> Adding index.md...")
+        with open(f"{folder_name}/poketes/index.md", 'w') as file:
+            file.write(Wiki.poketes(page_mode='index'))
+        for typ in types:
+            with open(f"{folder_name}/poketes/{typ}.md", 'w') as file:
+                file.write(Wiki.poketes(page_mode='multi', pokete_type=typ))
+
+        if quite or verbose:
+            print("==> Adding attacks...")
+        for page in Wiki.attacks(multi_page=True):
+            file_name, file_contents = page
+            with open(f"{folder_name}/attacks/{file_name}", 'w') as file:
+                file.write(file_contents)
+
+        if quite or verbose:
+            print("==> Adding types...")
+        with open(f"{folder_name}/types.md", 'w') as file:
+            file.write(Wiki.types(multi_page=True))
+
+        if quite or verbose:
+            print("==> Adding items...")
+        with open(f"{folder_name}/items.md", 'w') as file:
+            file.write(Wiki.items(multi_page=True))
+
+        if quite or verbose:
+            print("==> Adding effects...")
+        with open(f"{folder_name}/effects.md", 'w') as file:
+            file.write(Wiki.effects(multi_page=True))
+
+
+def gen_pics():
+    """The function to generate a markdown file with some example pictures."""
+    if quite or verbose:
+        print(":: Generating pics.md...")
+    md_str = "# Example pictures\n"
+    md_str += str.join("\n\n", [f"![{i}](ss/{i})" for i in sorted(os.listdir("assets/ss"))])
+
+    # writing to file
+    with open("assets/pics.md", "w+") as file:
+        file.write(md_str)
+
+
+if __name__ == "__main__":
+    if len(sys.argv) == 1:
+        silent, quite, verbose = False, True, False
```

Comment:
Looks like only one of those three is true at the same time, so why don't you just use a string or even an enum?
(P.S. it's called quiet)

Reply:
I didn't to type

```python
if verbosity in ["quiet", "verbose"]:
```

all the time, because using 

```python
if verbose or quiet:
```

is much easier (to read).